pkg/network/aci_network.go

```diff
@@ -42,36 +43,49 @@ type ProviderNetwork struct {
 }
 
 func (pn *ProviderNetwork) SetVNETConfig(ctx context.Context, azConfig *auth.Config) error {
+	ctx, span := trace.StartSpan(ctx, "network.SetVNETConfig")
+	defer span.End()
+
+	if azConfig.AKSCredential != nil {
```

Comment:
Is this a new behavior?

Reply:
No, this is only to add a trace for the 'SetVNETConfig' method.